docs/platform-services/automation-service/app-central/integrations/google-chat.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Google Chat
+description: ''
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('/img/platform-services/automation-service/app-central/logos/google.png')} alt="google" width="80"/>
+
+**Version: 2.0  
+Updated: August 27, 2024**
```

Comment:
is this change correct?

```suggestion
***Version: 2.0  
Updated: August 27, 2024**
```

Reply:
Thanks for catching that, @kimsauce. I changed it so that part is italicized and is consistent with the other App Central integration articles.